docs/reference/release-notes/8.14.3.asciidoc

```diff
@@ -0,0 +1,23 @@
+[[release-notes-8.14.3]]
+== {es} version 8.14.3
+
+
+Also see <<breaking-changes-8.14,Breaking changes in 8.14>>.
+
+[[bug-8.14.3]]
```

Comment:
```suggestion
[[known-issues-8.14.3]]
[float]
=== Known issues
* When upgrading clusters from version 8.11.4 or earlier, if your cluster contains non-master-eligible nodes,
information about the new functionality of these upgraded nodes may not be registered properly with the master node.
This can lead to some new functionality added since 8.12.0 not being accessible on the upgraded cluster.
If your cluster is running on ECK 2.12.1 and above, this may cause problems with finalizing the upgrade.
To resolve this issue, perform a rolling restart on the non-master-eligible nodes once all Elasticsearch nodes
are upgraded. This issue is fixed in 8.15.0.

[[bug-8.14.3]]
```